### ISSUE TITLE
Update alt text for peace-justice image on the About Us page to adhere to WCAG 

### ISSUE THREAD

**Issue by Wny-Duong** [hackforla/website#3400]
### Overview
We need to ensure that alt text for images adhere to Web Content Accessibility Guidelines (WCAG). In this issue, we will change the alt text for the peace-justice image on the [About Us page](https://www.hackforla.org/about/).

### Details
Designers and PMs identified that some alt texts in the Events page should be changed because they are either inaccurate or too long. Ideally, alt text should be descriptive but short and succinct. According to the WCAG page, an accompanying alt text should be short. Additionally, the Functional Accessibility Evaluator (FAE) recommends for alt text to be no more than 100 characters.

### Action Items
- [x] In the file `_includes/about-page/about-card-sustainability.html`, change the following alt text

From
```
Peace, Justice and Strong Intitutions goal logo
```

To
```
16 Peace, Justice and Strong Institutions
```

- [x] **For merge team**: Once the pull request associated with this issue is approved and merged, please update and edit epic #3361 by
  - [x] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #3361 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is part of epic #3361 
- [About Us page](https://www.hackforla.org/about/)
- [Alt text decision Tree ](https://www.w3.org/WAI/tutorials/images/decision-tree/)
- [FAE rule 4: alt text should be less than 100 characters](https://fae.disability.illinois.edu/rulesets/IMAGE_4_EN/)

#### Information about WCAG
- [WCAG](https://www.w3.org/WAI/standards-guidelines/wcag/)
- [Web Accessibility Tutorials: Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)
- [WCAG section on alt text](https://www.w3.org/WAI/WCAG21/Techniques/html/H37.html)

**Comment by Mattre7** [comment]
i. Availability: Available to work tonight, Friday-Sunday I will be able to work in the mornings.
ii. ETA: Likely can have this done by tonight, should not take long

**Comment by Mattre7** [comment]
Caught a spelling error in the instructions above, Intitutions should be Institutions, made that change in my commit. 

Pull request is now opened and pending review. Only one file change with no other conflicts
